src/main/java/com/airlines/dao/impl/CrewMemberDaoImpl.java

```diff
@@ -102,4 +103,27 @@ public CrewMember findById(int id) {
             throw new DaoOperationException(String.format("Error finding crew member by id = %d", id), e);
         }
     }
+
+    @Override
+    public void addCrewMemberToCrew(CrewMember crewMember, int crewID) {
+        Objects.requireNonNull(crewMember);
+
+        if (crewMember.getId() == 0) {
+            throw new DaoOperationException("Cannot find a crew member without ID");
+        }
+
+        try (Connection connection = dbConnector.getDBConnection();
+             PreparedStatement updateStatement = connection.prepareStatement(UPDATE_CREW_ID_QUERY)) {
+
+            updateStatement.setInt(1, crewID);
+            updateStatement.setInt(2, crewMember.getId());
+            int rowsAffected = updateStatement.executeUpdate();
+
+            if (rowsAffected == 0) {
+                throw new DaoOperationException(String.format("Crew member with id = %d does not exist", crewMember.getId()));
```

Comment:
I would rather wrote:
Crew update for crew member with id = %d was not performed 
smth like that.
Because I am not 100% sure, that update database row was not performed only because user doesn't exist.

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/80c1f66083580a28c27bea038eea1e050a97f30e